web/libs/datamanager/src/components/Filters/Filters.scss

```
@@ -0,0 +1,85 @@
+.filters {
+  background-color: white;
+  position: relative;
+
+  &:not(&_sidebar) {
+    padding-top: 10px;
+    min-width: 400px;
+    border-radius: 2px;
+    box-shadow: 0 3px 6px -4px rgba(0, 0, 0, 0.12),
+      0 6px 16px 0 rgba(0, 0, 0, 0.08), 0 9px 28px 8px rgba(0, 0, 0, 0.05);
```

Flagging as opportunity for css var replacement. Further usage review needed.
```suggestion
    box-shadow: 0 3px 6px -4px rgba(var(--black), 0.12),
      0 6px 16px 0 rgba(rgba(--black), 0.08), 0 9px 28px 8px rgba(var(--black), 0.05);
```